src/main/java/com/wise23/chariteed/ChariteEdApplication.java

```diff
@@ -6,8 +6,9 @@
 @SpringBootApplication
 public class ChariteEdApplication {
 
+	public static String server_url = "https://hapi.fhir.org/baseR4";
```

Comment:
Not the best place for the url adress. I would suggest to move it to application.properties

Reply:
When I move these URLs to application.properties and then try to access them, I always get `null`

For example:

`@Value("${testserver.url}")
  private String url;`

`url` is in my case always `null`. Why is that?

Reply:
@JannesVolkens no idea, maybe url is a reserved word?

Reply:
Done